### DISCUSSION TITLE
Navidrome login

### DISCUSSION THREAD

**Discussion by unecc** [tteck/Proxmox#3352] [Help Needed]
Hi, i would try navidrome but i don't know user and password for login.

Where can i find it?

**Comment by tteck** [comment]
You have to create a user/password

![Screenshot from 2024-07-09 06-45-01](https://github.com/tteck/Proxmox/assets/21342670/f2986c22-9dcb-4fd3-8bd2-042baf4b2778)

**Reply by unecc** [reply]
no, i've used this script on proxmox



bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/navidrome.sh)"

**Reply by tteck** [reply]
so did I

**Reply by tteck** [reply]
After creation, you go to IP:4533 and the screen above is what shows

**Reply by unecc** [reply]
![image](https://github.com/tteck/Proxmox/assets/134554559/f128f3cb-69fd-420d-95ef-b4f68710de43)

Oh maybe i've to access to web interface for set login?

![image](https://github.com/tteck/Proxmox/assets/134554559/33654699-9d78-49bc-8130-4881356980c2)

**Reply by tteck** [reply]
Are you using SSH? ~Must have something blocking http://192.186.1.91:4533 🤷🏻‍♂️~

**Comment by tteck** [comment]
Oh I see that it didn't execute the `navidrome-install.sh` How are you trying to run the script?

**Reply by unecc** [reply]
I'm using this script in proxmox.
node/shell/ 
The script is wrong?

**Reply by tteck** [reply]
No, it's working correctly on this end

```
    _   __            _     __                        
   / | / /___ __   __(_)___/ /________  ____ ___  ___ 
  /  |/ / __  / | / / / __  / ___/ __ \/ __  __ \/ _ \
 / /|  / /_/ /| |/ / / /_/ / /  / /_/ / / / / / /  __/
/_/ |_/\__,_/ |___/_/\__,_/_/   \____/_/ /_/ /_/\___/ 
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: navidrome
Using Disk Size: 4
Allocated Cores: 2
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Navidrome LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.112.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Navidrome
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Navidrome should be reachable by going to the following URL.
         http://192.168.86.250:4533 
```

**Reply by tteck** [reply]
Looks like it can't set locale for some reason. That's why I asked if you were using SSH.
